src/components/CommentTree/comments.js

```diff
@@ -0,0 +1,82 @@
+const profilePicture = '/static/profilePicture1.png'
+
+export const mkComment = (n, children) => ({
+  id: n,
+  displayAuthor: {
+    profilePicture,
+    name: `${n} – Paul Ullrich`,
```

Comment:
I'd vote again for less controversial dummy content, e.g. use
name: Christof Moser
credential: Journalist
content: 'Journalismus strebt nach Klarheit, er ist der Feind der uralten Angst vor dem Neuen.'
(or something else from the Republik website)